smoke-tests/designer/features/pageobjects/pages/formRunner.page.js

```diff
@@ -53,10 +53,23 @@ class FormRunnerPage extends Page {
    * @param fieldValue
    */
   inputField(labelText, fieldValue) {
-    let fieldIndex = browser
-      .$$(".govuk-input")
-      .findIndex((el) => el.parentElement().getText().includes(labelText));
-    return browser.$$(".govuk-input")[fieldIndex].setValue(fieldValue);
+    const inputElement = this.getInputFieldByLabelText(labelText);
+    inputElement.waitForDisplayed(); // Wait until the input field is displayed
+    inputElement.setValue(fieldValue);
+  }
+
+  getInputFieldByLabelText(labelText) {
+    const inputFields = browser.$$(".govuk-input");
```

Comment:
Just a suggestion, maybe use a data attribute instead of using a class as the locator as classes aren't usually very robust. @adamdavies1